test/parallel/test-process-argv-0.js

```diff
@@ -14,15 +9,10 @@ if (process.argv[2] !== 'child') {
   });
 
   var childArgv0 = '';
-  var childErr = '';
   child.stdout.on('data', function(chunk) {
     childArgv0 += chunk;
   });
-  child.stderr.on('data', function(chunk) {
-    childErr += chunk;
-  });
   child.on('exit', function() {
```

Comment:
I don't see how `common.mustCall` can be used to replace the assertion here. Can you explain a bit more what you mean?